tests/e2e/create/create_test.go

```diff
@@ -218,6 +219,10 @@ var _ = Describe("Test 'create' command", Ordered, func() {
 			Expect(descriptor).ToNot(BeNil())
 			Expect(descriptor.SchemaVersion()).To(Equal(v2.SchemaVersion))
 
+			descriptorSplit := strings.Split(descriptor.Name, "/")
+			descriptorName := descriptorSplit[len(descriptorSplit)-1]
+			Expect(template.Name).To(Equal(descriptorName + "-" + descriptor.Version))
```

Comment:
```suggestion
			Expect(template.Name).To(Equal("template-operator-1.0.0"))
```

Reply:
There is no need to get the name from the descriptor. Also can you add this line to some of the tests below as well?